packages/babel-parser/src/index.js

```diff
@@ -98,3 +98,6 @@ function getParserClass(pluginsFromOptions: PluginList): Class<Parser> {
   }
   return cls;
 }
+
+export default parse;
+export { parse };
```

Comment:
Nit: Babel parser also exposes `parseExpression`, consider merge the export declaration here.

Reply:
Wouldn't it be better to keep the `export` on function definition so that you don't need to repeat exported names?